tock/tock/templates/base.html

```diff
@@ -12,6 +12,8 @@
     {% else %}
     <link rel="shortcut icon" href="{% static 'img/favicon-happy.png' %}">
     {% endif %}
+    <meta name="tock-icon-happy" content="{% static 'img/favicon-happy' %}">
+    <meta name="tock-icon-angry" content="{% static 'img/favicon-angry' %}">
```

Comment:
curious: why are these custom meta elements needed?

Reply:
from 0b361ebff4b918063d0a40df27c04f2c93d2066d
> Add meta tags with icons;
> This is possibly helpful for future iterations on the Tock notification
work which is comprised of JS rather than Python. We'd have to really
refactor the Notification code to be aware of users when they're late to
properly swap the icon. But for now, the icon will default to whatever
is on the page when the user first loads it.

My original intention was to have these get pulled in from the JS code and leveraged whenever the notification timer would run, but the current implementation of the notification code relies on time rather than actually checking if the user is late or not. So without building an API endpoint for that now, I'm leaving the `<meta>` tags here in case someone wants to implement it in the future, myself or whomever. For now it's useful for the the notification code to pull from the `<link>` tag as it's mostly accurate. But by the time the notification actually goes out, the user may or may not be late.

This PR though I think achieves my original intent with these changes. It's all to make Tock less aggressive about reminding folks to Tock their time. AngryTock is a nice tool when it's needed, but a majority of folks aren't late and it's a bit overly aggressive to only have AngryTock be the iconography for the site. Even if it is just used in the Favicon and Notifications for now.

Reply:
The notification code is intended as a reminder for later in the day on Friday so folks don't forget to Tock their time on time.  If anything, this would be a slight enhancement to the feature to also notify the user that their timecard is actually late come the following week.

I think we're at a good place to stop here, and a future PR can address this tweak in functionality and add the necessary enhancements. :-)

/cc @mgwalker since he put the original notifications in place and may be interested to see all of this work and discussion!